kong/llm/drivers/shared.lua

```diff
@@ -719,7 +725,10 @@ function _M.post_request(conf, response_object)
     meta_container[log_entry_keys.LLM_LATENCY] = llm_latency
 
     if response_object.usage and response_object.usage.completion_tokens then
-      local time_per_token = math.floor(llm_latency / response_object.usage.completion_tokens)
+      local time_per_token = ((llm_latency > 0) and
+                             (response_object.usage.completion_tokens > 0) and
+                             math.floor(llm_latency / response_object.usage.completion_tokens))
+                             or 0
```

Comment:
let's avoid using one-liners but replace with proper if else block for readability.
```suggestion
      local time_per_token = 0
      if response_object.usage.completion_tokens > 0 then
        time_per_token  = math.floor(llm_latency / response_object.usage.completion_tokens)
      end
```